engine/src/main/java/org/terasology/engine/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -108,75 +122,101 @@ public class UniverseSetupScreen extends CoreScreenLayer implements UISliderOnCh
+     * Initialise and populate screen and add event handlers to control elements.
+     * Screen elements (x - control elements):
+     * - (x) World Generator Dropdown
+     * - (x) Seed Display & Edit Field
+     * - (x) Seed Re-Roll Button
+     * - Preview Display
+     * - (x) Zoom Slider
+     * - multiple text fields
+     * - (x) return button
+     * - (x) exit button
+     * - (x) start button
+     */
     @Override
     public void initialise() {
         setAnimationSystem(MenuAnimationSystems.createDefaultSwipeAnimation());
 
+        /*
+         * initialize world generator dropdown:
+         * - bind options read-only
+         * - set visible options & rendering
+         * - bind selection read & write
+         */
         final UIDropdownScrollable<WorldGeneratorInfo> worldGenerators = find("worldGenerators", UIDropdownScrollable.class);
-        if (worldGenerators != null) {
-            worldGenerators.bindOptions(new ReadOnlyBinding<List<WorldGeneratorInfo>>() {
-                @Override
-                public List<WorldGeneratorInfo> get() {
-                    // grab all the module names and their dependencies
-                    // This grabs modules from `config.getDefaultModSelection()` which is updated in AdvancedGameSetupScreen
-                    final Set<Name> enabledModuleNames = new HashSet<>(getAllEnabledModuleNames());
-                    final List<WorldGeneratorInfo> result = Lists.newArrayList();
-                    for (WorldGeneratorInfo option : worldGeneratorManager.getWorldGenerators()) {
-                        //TODO: There should not be a reference from the engine to some module.
-                        //      The engine must be agnostic to what modules may do.
-                        if (enabledModuleNames.contains(option.getUri().getModuleName())
-                                && !option.getUri().toString().equals("CoreWorlds:heightMap")) {
-                            result.add(option);
-                        }
+        worldGenerators.bindOptions(new ReadOnlyBinding<List<WorldGeneratorInfo>>() {
+            @Override
+            public List<WorldGeneratorInfo> get() {
+                // grab all the module names and their dependencies
+                // This grabs modules from `config.getDefaultModSelection()` which is updated in AdvancedGameSetupScreen
+                final Set<Name> enabledModuleNames = new HashSet<>(getAllEnabledModuleNames());
+                final List<WorldGeneratorInfo> result = Lists.newArrayList();
+                for (WorldGeneratorInfo option : worldGeneratorManager.getWorldGenerators()) {
+                    //TODO: There should not be a reference from the engine to some module.
+                    //      The engine must be agnostic to what modules may do.
+                    if (enabledModuleNames.contains(option.getUri().getModuleName())
+                            && !option.getUri().toString().equals("CoreWorlds:heightMap")) {
```

Comment:
didn't find any, so I guess "no"?
I'm not sure about that use case, but maybe I don't fully grasp it? Can you elaborate a bit on what you're imagining and in which situations it might be useful?